src/client/canvas/svg/group.tsx

```diff
@@ -0,0 +1,20 @@
+import { observer } from 'mobx-react'
+import * as React from 'react'
+import { Component } from 'react'
+
+import { Transform } from '../../math/transform'
+import { Group as GroupGeometry } from '../object/group'
+import { Shape } from '../object/shape'
+
+import { transform, viewForGeometry } from './svg'
+
+@observer
+export class Group extends Component<{model: GroupGeometry, world: Transform}> {
+
+  public render() {
+    return <g transform={transform(this.props.model.transform)}>
+      {this.props.model.children.map((obj, i) =>
+        viewForGeometry(obj, i, this.props.world.clone().then(this.props.model.transform.inverse())))}
+    </g>
```

Comment:
⛳️ ing:

```ts
type Props = { model: GroupGeometry, world: Transform }
export const Group = observer(({ model: { children, transform }, world }: Props) => (
  <g transform={toSvgTransform(transform)}>
    {children.map((child, i) => viewForGeometry(child, i, world.clone().then(transform)))}
  </g>
))
```

We can apply this to every react component, since they are all stateless.